tools/github-actions/audit/src/reports.ts

```diff
@@ -0,0 +1,179 @@
+import * as core from '@actions/core';
+import {
```

Comment:
can these be type imports?

Reply:
not sure as we import namespaces and not typings